x-pack/plugin/esql/qa/testFixtures/src/main/resources/qualifiers.csv-spec

```diff
@@ -0,0 +1,167 @@
+basic
+required_capability: name_qualifiers
+
+FROM sample_data main
```

Comment:
We may want to require the `AS` keyword, at least in the beginning - if we allow qualifiers in the `FROM` clause at all:
```
FROM sample_data AS main
```